src/doc/index.md

```diff
@@ -63,9 +63,10 @@ fn main() {
 
 Cargo generated a “hello world” for us. Let’s compile it:
 
-<pre><code class="language-shell">$ cargo build
-<span style="font-weight: bold"
-class="s1">   Compiling</span> hello_world v0.1.0 (file:///path/to/project/hello_world)</code></pre>
+```shell
```

Comment:
This and similar changes (replacing code block with triple-backticks) are to be able to keep the content in sync. We are loosing some styling in the `shell` snippets, but it's fine, since we will used them eventually anyway.